go/pb/lmsysmd/load/data/v1/data.go

```diff
@@ -0,0 +1,103 @@
+package data
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"sync"
+
+	"connectrpc.com/connect"
+	datav1 "github.com/Lev1ty/lmsysmd/pbi/lmsysmd/load/data/v1"
+	"github.com/Lev1ty/lmsysmd/pbi/lmsysmd/load/data/v1/datav1connect"
+	"github.com/jackc/pgx/v5/pgxpool"
+
+	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/jwt"
+	"google.golang.org/api/drive/v3"
+	"google.golang.org/api/option"
+	"google.golang.org/api/sheets/v4"
+)
+
+type DataService struct {
+	datav1connect.UnimplementedDataServiceHandler
+	dbOnce sync.Once
+	db     *pgxpool.Pool
+	gOnce  sync.Once
+	gsSrv  *sheets.Service
+	gdSrv  *drive.Service
+	gConf  *jwt.Config
+}
+
+var re = regexp.MustCompile(`https:\/\/docs\.google\.com\/spreadsheets\/d\/([a-zA-Z0-9_-]+)\/`)
+
+var caseCategoryEnumMap = map[string]datav1.Data_CaseCategory{
+	"BR": datav1.Data_CASE_CATEGORY_BREAST,
```

Comment:
I don't think you need to add explicit validation since we are already using buf validate

Reply:
I have to check for errors anyway when I try to read a map for a key that doesn't exist so I will return an error there otherwise will leave it to buf validate